src/contents/all/components/app/App.tsx

```diff
@@ -97,6 +97,28 @@ const App = () => {
     })
   }
 
+  const deleteKnot = (knot) => {
```

Comment:
so I'm declaring deleteKnot and editKnot in App.tsx, and am passing them to Knots.tsx, then Knot.tsx. Lmk if this flow is wrong.

Reply:
hmm after looking through it I'm thinking might want to move these into `Knot.tsx` so we don't have to pass it down props. Since the only place we'll need edit/delete functionality is an existing knot. We'll probably want to leverage `KnotInput` in Knot as well for edit, then possibly pass that function into `KnotInput` as a prop.